### PR TITLE
[plugin.audio.navahang] v2.1.0

### PR DESCRIPTION
**Pull request by amirrammari**
### Checklist:

- [x] My code follows the [add-on rules](http://kodi.wiki/view/Add-on_rules) and [piracy stance](http://kodi.wiki/view/Official:Forum_rules#Piracy_Policy) of this project. 
- [x] I have read the [CONTRIBUTING](https://github.com/xbmc/repo-plugins/blob/master/CONTRIBUTING.md) document
- [x] Each add-on submission should be a single commit with using the following style: [plugin.video.foo] v1.0.0

Additional information :
- Submitting your add-on to this specific branch makes it available to any Kodi version equal or higher than the branch name with the applicable Kodi dependencies limits.
- [add-on development](http://kodi.wiki/view/Add-on_development) wiki page.
- Kodi [pydocs](http://kodi.wiki/view/PyDocs) provide information about the Python API
- [PEP8](https://www.python.org/dev/peps/pep-0008/) codingstyle which is considered best practice but not mandatory.
- This add-on repository has automated code guideline check which could help you improve your coding. You can find the results of these check at [Codacy](https://www.codacy.com/app/Kodi/repo-plugins/dashboard). You can create your own account as well to continuously monitor your python coding before submitting to repo.
- Development questions can be asked in the [add-on development](http://forum.kodi.tv/forumdisplay.php?fid=26) section on the Kodi forum.

### CODE REVIEW

**Comment by Rechi** [comment]
Please try to update this PR instead of openenig a new one, this time. A new PR is annoying as you loose the already existing comments.

File: plugin.audio.navahang/default.py [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+from xbmcplugin import addDirectoryItem, endOfDirectory
+
+ADDON = xbmcaddon.Addon()
+ADDONNAME = ADDON.getAddonInfo('name')
+ADDONID = ADDON.getAddonInfo('id')
+ADDONVERSION = ADDON.getAddonInfo('version')
+CWD = ADDON.getAddonInfo('path').decode("utf-8")
+LANGUAGE = ADDON.getLocalizedString
+
+DataLink = 'http://navahang.com/~siavash/webservice/GetMp3?type=Featured&from=0&to=20'
```
[the remark this answers is not in the document]
Comment (amirrammari) [reply plugin.audio.navahang/default.py]: Although it looks like this URL is not official, I got it from the site admin.
Please see the URL domain, it's official (http://navahang.com)

[the remark this answers is not in the document]
Comment (Rechi) [reply plugin.audio.navahang/default.py]: I know it's under the official domain, but the path (`/~`) looks a bit suspicious to me.

**Comment by Rechi** [comment]
> Each add-on submission should be a single commit with using the following style: [plugin.video.foo] v1.0.0

Please squash the commits.

File: plugin.audio.navahang/default.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (amirrammari) [reply plugin.audio.navahang/default.py]:
```suggestion
DataLink = 'http://navahang.com/webservice/GetMp3?type=Featured&from=0&to=20'
```

[the remark this answers is not in the document]
Comment (Rechi) [reply plugin.audio.navahang/default.py]: If that url works too, I suggest using that.

[the remark this answers is not in the document]
Comment (amirrammari) [reply plugin.audio.navahang/default.py]: In my country, singing women is a crime and women can't singing officially in Iran.
That's why most music sites get blocked.
Now the com domain blocked and only me domain working in my country.

[the remark this answers is not in the document]
Comment (amirrammari) [reply plugin.audio.navahang/default.py]:
```suggestion
DataLink = 'http://navahang.com/webservice/GetMp3?type=Featured&from=0&to=20'
```

[the remark this answers is not in the document]
Comment (amirrammari) [reply plugin.audio.navahang/default.py]:
```suggestion
DataLink = 'http://navahang.com/webservice/GetMp3?type=Featured&from=0&to=20'
```

[the remark this answers is not in the document]
Comment (amirrammari) [reply plugin.audio.navahang/default.py]:
```suggestion
DataLink = 'http://navahang.com/webservice/GetMp3?type=Featured&from=0&to=20'
```

**Comment by amirrammari** [comment]
```suggestion
DataLink = 'http://navahang.com/webservice/GetMp3?type=Featured&from=0&to=20'
```

File: plugin.audio.navahang/default.py [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+from xbmcplugin import addDirectoryItem, endOfDirectory
+
+ADDON = xbmcaddon.Addon()
+ADDONNAME = ADDON.getAddonInfo('name')
+ADDONID = ADDON.getAddonInfo('id')
+ADDONVERSION = ADDON.getAddonInfo('version')
+CWD = ADDON.getAddonInfo('path').decode("utf-8")
+LANGUAGE = ADDON.getLocalizedString
+
+DataLink = 'http://navahang.me/webservice/GetMp3?type=Featured&from=0&to=20'
```
Comment (amirrammari) [inline plugin.audio.navahang/default.py]:
```suggestion
DataLink = 'http://navahang.com/webservice/GetMp3?type=Featured&from=0&to=20'
```

Comment (amirrammari) [reply plugin.audio.navahang/default.py]: @Rechi I changed .me domain to official domain (.com domain)
Please check this plugin again and report problems.

**Comment by amirrammari** [comment]
@Rechi Hello, What should I do?

**Comment by Rechi** [comment]
> Each add-on submission should be a single commit with using the following style: [plugin.video.foo] v1.0.0

Please squash the commits.

**Comment by amirrammari** [comment]
@Rechi How I can squash the commits? I don't have enough information to do this.

**Comment by Rechi** [comment]
To fix your pull request execute the following commands (origin is your repo, upstream is the xbmc repo)

```
git fetch upstream
git checkout krypton
git reset --hard upstream/krypton
git checkout origin/krypton -- plugin.audio.navahang/
git commit -m '[plugin.audio.navahang] v2.1.0'
git push -f origin krypton
```

**Comment by amirrammari** [comment]
Must I download the whole "repo-plugins" to my computer?
image

**Comment by Rechi** [comment]
At least you have to fetch the base branch (krypton of xbmc/repo-plugins) and the target branch (kyrpton of  amirrammari/repo-plugins).

**Comment by amirrammari** [comment]
@Rechi 
I will close this pull request
Please add my add-on by yourself:
https://github.com/amirrammari/kodi-navahang-addon
Thanks.

**Pull request closed without merging**